package.json

```diff
@@ -141,5 +142,12 @@
     "not ie < 11",
     "not op_mini all"
   ],
-  "dependencies": {}
+  "resolutions": {
```

Comment:
These are not strictly needed, but fix the build to avoid duplicated package imports with just differing versions.

Reply:
how did you figure this out?

Reply:
I looked for node_modules directories inside the packages/* containing various packages, and then chose a version of the packages for here.  That forces a unified version.  In the case of uuid I chose a version based on the comments on earlier versions being problematic, while for core-js, I chose a version that allows the istanbul instrumenter to run (which it does not on core-js v3).  Canvas was chosen based on being compatible with node 18 and having pre-built binaries for it.